dm/worker/server.go

```diff
@@ -147,7 +149,10 @@ func (s *Server) Start() error {
 			log.L().Error("fail to start gRPC server", log.ShortError(err2))
 		}
 	}()
-	go InitStatus(httpL) // serve status
+
+	go initStatusOnce.Do(func() {
```

Comment:
only need do once, may cause data race in unit test

Reply:
Does the unit test calls `Start` multiple times for the same `Server` instance?

Reply:
no, this update dosen't fix the data race. In fact it is conflict with etcd, you can look https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/dm_ghpr_test/detail/dm_ghpr_test/3961/pipeline/66/
I revert this commit and fix the real data race in 091fbe5bf819bad6fa2e8eaa0c5b42a88b7a6613